layouts/partials/profiling/profiling-languages.html

```diff
@@ -17,7 +17,12 @@
           {{ partial "img.html" (dict "root" . "src" "integrations_logos/golang.png" "class" "img-fluid" "alt" "Go" "width" "400") }}
         </div>
       </a>
-      </div>
+      <a class="card" href="/tracing/profiler/getting_started/?tab=ruby">
```

Comment:
```suggestion
      <a class="card" href="/tracing/profiler/getting_started/?code-lang=ruby">
```

(to match the above)